frontend/src/features/admin/users/constants/index.tsx

```diff
@@ -4,14 +4,15 @@ import { CellProps } from 'react-table';
 import { RowActions } from '../components/RowActions';
 import { IUserRecord } from '../interfaces/IUserRecord';
 import { FaCheck, FaTimes } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
 
 export const columnDefinitions: ColumnWithProps<IUserRecord>[] = [
   {
     Header: 'IDIR/BCeID',
     accessor: 'username',
     align: 'left',
     Cell: (props: CellProps<IUserRecord>) => {
-      return <a href={`/admin/user/${props.row.original.id}`}>{props.row.original.username}</a>;
+      return <Link to={`/admin/user/${props.row.original.id}`}>{props.row.original.username}</Link>;
```

Comment:
React router links work the same as a tags for navigation but the won't force the page to refresh.